pallets/automation-time/src/tests.rs

```diff
@@ -2234,13 +2271,19 @@ fn auto_compound_delegated_stake_without_minimum_balance() {
 
 		AutomationTime::trigger_tasks(Weight::from_ref_time(120_000));
 
-		events()
+		let emitted_events = events();
+
+		// Expected result:
+		// 1. The current execution will be skipped, triggering the emission of an AutoCompoundDelegatorStakeSkipped event, message: Balance less than minimum
+		emitted_events.clone().into_iter()
+			.find(|e| matches!(e, RuntimeEvent::AutomationTime(crate::Event::AutoCompoundDelegatorStakeSkipped { message, .. }) if *message == "Balance less than minimum".as_bytes().to_vec())).expect("AutoCompoundDelegatorStakeSkipped event should have been emitted");
+
```

Comment:
Is it easy for the tests.rs file to reference the error messages in lib.rs?

For example, here you can use `*message == ERROR_MESSAGE_BALANCE_LOW` to refer to the message definition in lib.rs.